mon-pix/app/templates/components/qroc-proposal.hbs

```diff
@@ -9,6 +9,7 @@
       <textarea class="challenge-response__proposal challenge-response__proposal--paragraph"
                 rows="5"
                 id="qroc_input"
+                onkeydown={{action 'onInputChange'}}
```

Comment:
Je me demande si on ne pourrait pas privilégier l'attribut [`@key-down`](https://guides.emberjs.com/release/components/built-in-components/#toc_actions). 

Après réflexion, je ne suis pas sûr qu'il soit compatible avec un composant builtin `textarea` 🤔

Reply:
Le component n'est pas encore glimmeriser, donc passer par @key-down pose actuellement quelques problèmes.
Je n'ai pas pris en compte ces modifications, mais ta remarque m'a permis de penser à rajouter des tests dans le cas des text-area, qui peut changer des choses.